phylogenetic/rules/prepare_sequences.smk

```diff
@@ -11,10 +11,38 @@ and will produce an aligned FASTA file of subsampled sequences as an output.
 """
 
 
+rule download:
+    output:
+        sequences="data/{virus}/sequences.fasta.zst",
+        metadata="data/{virus}/metadata.tsv.zst",
+    params:
+        sequences_url="https://data.nextstrain.org/files/workflows/seasonal-cov/sequences_{virus}.fasta.zst",
+        metadata_url="https://data.nextstrain.org/files/workflows/seasonal-cov/metadata_{virus}.tsv.zst",
```

Comment:
These should be updated to pull from the new `/{virus}/sequences.fasta.zst` + `/{virus}/metadata.tsv.zst` URLs.

Reply:
UGH (thank you for catching that) 

done!

Reply:
Hah, now the [CLI is failing](https://github.com/nextstrain/seasonal-cov/actions/runs/10065851680/job/27825897432) because the files don't exist 🙈

Reply:
_SIGHS HARDER_